mk/platform.mk

```diff
@@ -154,4 +154,4 @@ ${BIN_DIR}:
 $(TEST_OUT_DIR):
 	mkdir -p $(TEST_OUT_DIR)
 
--include $(DEPS)
```

Comment:
I think you only want this change for debugging, and shouldn't be checking it in.